tensorflow/lite/micro/tools/make/download_and_extract.sh

```diff
@@ -185,6 +185,14 @@ patch_cmsis() {
     -iname '*.*' -exec \
     sed -i -E $'s@#include "dsp/matrix_functions.h"@#include "cmsis/CMSIS/DSP/Include/dsp/matrix_functions.h"@g' {} \;
 
+  find tensorflow/lite/micro/tools/make/downloads/cmsis \
+    -iname '*.*' -exec \
+    sed -i -E $'s@#include "cmsis_compiler.h"@#include "cmsis/CMSIS/Core/Include/cmsis_compiler.h"@g' {} \;
+
+  find tensorflow/lite/micro/tools/make/downloads/cmsis \
+    -iname '*.*' -exec \
+    sed -i -E $'s@#include "arm_helium_utils.h"@#include "cmsis/CMSIS/DSP/Include/arm_helium_utils.h"@g' {} \;
+
```

Comment:
I'm guessing https://github.com/tensorflow/tensorflow/pull/44014 will make this change not be necessary anymore? That should get merged today.

Reply:
Ok, removing it